### PR TITLE
Removed subflows

### PR DESCRIPTION
**Pull request by VaniHaripriya**

### CODE REVIEW

File: src/main/resources/pipeline.sw.yaml [hunk 1]
```diff
@@ -26,7 +26,13 @@ states:
...
     type: operation
     actions:
-      - subFlowRef: pre_processing
+      - functionRef:
+          refName: python
+          arguments:
+            script: "kserve_payload = to_kserve(image)"
+            image: ".image"
+    stateDataFilter:
+      output: "{kserve_payload:$WORKFLOW.python.kserve_payload}"
```
Comment (hbelmiro) [inline src/main/resources/pipeline.sw.yaml]:
```suggestion
      output: "{kserve_payload:$WORKFLOW.python.kserve_payload, image: .image}"
```

File: src/main/resources/pipeline.sw.yaml [hunk 2]
```diff
@@ -41,7 +47,13 @@ states:
...
     type: operation
     actions:
-      - subFlowRef: post_processing
+      - functionRef:
+          refName: python
+          arguments:
+            script: "output_image = handle_response(image, kserve_response, output_image)"
+            image: ".image"
+            kserve_response: ".kserve_response"
+            output_image: ".output_image.jpg"
```
Comment (hbelmiro) [inline src/main/resources/pipeline.sw.yaml]:
```suggestion
            output_image: "./output_image.jpg"
```

File: src/main/resources/pipeline.sw.yaml [hunk 3]
```diff
@@ -41,7 +47,13 @@ states:
...
     stateDataFilter:
       output: '{output_image: .output_image}'
```
Comment (hbelmiro) [inline src/main/resources/pipeline.sw.yaml]:
```suggestion
      output: '{output_image: $WORKFLOW.python.output_image}'
```

**Review by hbelmiro — COMMENTED** [review]
@VaniHaripriya my suggestions should make the test pass.

**Comment by VaniHaripriya** [comment]
> @VaniHaripriya my suggestions should make the test pass.

@hbelmiro Thank you Helber, I will try it.

**Review by hbelmiro — APPROVED** [review]

**Pull request merged**